pkg/domain/infra/abi/network.go

```diff
@@ -115,6 +116,14 @@ func (ic *ContainerEngine) NetworkCreate(ctx context.Context, name string, optio
 		err      error
 		fileName string
 	)
+	config, err := ic.Libpod.GetConfig()
+	if err != nil {
+		return nil, err
+	}
+	// Acquire a lock for CNI
+	if _, err := storage.GetLockfile(filepath.Join(config.Engine.TmpDir, network.LockFileName)); err != nil {
```

Comment:
This does not yet acquire the lock.

`storage.GetLockfile` returns a `Locker, error` and we currently ignore the `Locker`.

To acquire the lock, we need to:
```golang
lock, err := storage.GetLockfile(...)
...
lock.Lock()
defer lock.Unlock()
```